### PR TITLE
[RFC][PL-893] Create base cmd

### PR DESCRIPTION
**Pull request by hfcorreia**

### CODE REVIEW

File: pkg/cmd/copytable/copytable.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+func New(config *viper.Viper) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:       fmt.Sprintf("%s <source-table> <target-table>", cmdName),
+		Short:     shortDescription,
+		Args: cobra.ExactArgs(2),
+		Run: func(cmd *cobra.Command, args []string) {
+		},
+	}
+
+	cmd.Flags().StringP("source-region", "s", "eu-west-1", "Set the AWS region for the source table")
```
Comment (jkrnak) [inline pkg/cmd/copytable/copytable.go]: In the future, default perhaps could be read from env variable `AWS_DEFAULT_REGION`

Comment (hfcorreia) [reply pkg/cmd/copytable/copytable.go]: My idea is for this cmd to be a wrapper around the sdk. So it should use the sdk's env vars.

File: Gopkg.toml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[[constraint]]
```
Comment (jmdalmeida) [inline Gopkg.toml]: Completely optional, but I think this project could be good to start trying out go modules. What do you think? 😄

File: pkg/cmd/copytable/copytable.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+const (
+	cmdName = "copy-table"
+
```
Comment (raffo0707) [inline pkg/cmd/copytable/copytable.go]: why empty line?

File: pkg/cmd/copytable/copytable.go [hunk 4]
```diff
@@ -0,0 +1,34 @@
+package copytable
```
Comment (jmdalmeida) [inline pkg/cmd/copytable/copytable.go]: From what I've read about project structures, the `pkg` subfolder is usually reserved for code meant to be imported in other projects. Also, I think you should follow cobra's project structure in this case (refer to: https://github.com/spf13/cobra#getting-started).

Comment (hfcorreia) [reply pkg/cmd/copytable/copytable.go]: that's the idea. we will need to do that for our private project

**Pull request merged**